cmd/lakectl/cmd/annotate.go

```diff
@@ -29,23 +29,36 @@ var annotateCmd = &cobra.Command{
 	Args:    cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		pathURI := MustParsePathURI("path", args[0])
+		recursive, _ := cmd.Flags().GetBool("recursive")
 		client := getClient()
 		pfx := api.PaginationPrefix(*pathURI.Path)
 		context := cmd.Context()
 		resp, err := client.ListObjectsWithResponse(context, pathURI.Repository, pathURI.Ref, &api.ListObjectsParams{Prefix: &pfx})
 		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusOK)
+		const delimiter = "/"
+		var paramsDelimiter api.PaginationDelimiter
+		if recursive {
+			paramsDelimiter = ""
+		} else {
+			paramsDelimiter = delimiter
+		}
```

Comment:
`paramsDelimiter` is already an empty string no?

```suggestion
		if !recursive {
			paramsDelimiter = delimiter
		}
```